conformance/smart-config-fhirr4-service/README.md

```diff
@@ -1 +1,61 @@
 # FHIR R4 SMART Configuration Service
+
+## Introduction
+
+This service provides API implementation of 
+[SMART Configuration API](https://www.hl7.org/fhir/smart-app-launch/#discovery-of-server-capabilities-and-configuration) 
+of a FHIR server.
+
+SMART defines a discovery document available at `.well-known/smart-configuration` relative to a FHIR Server Base URL, 
+allowing clients to learn the authorization endpoint URLs and features a server supports. This information helps clients to 
+direct authorization requests to the right endpoint, and helps clients construct an authorization request that the server 
+can support.
+
+```Supported FHIR version is 4.0.1.```
+
```

Comment:
Concerns on https://github.com/wso2/open-healthcare-prebuilt-services/pull/42/files#r1421948514 https://github.com/wso2/open-healthcare-prebuilt-services/pull/42/files#r1421949674 same for this service as well.

Reply:
Updated in https://github.com/wso2/open-healthcare-prebuilt-services/commit/8b982af37b97ff9cd7f6f9e8a273a8c6a9400010